README.md

```diff
@@ -141,7 +141,18 @@ I do try to be responsive to PRs though, so if you leave one I'll try to get it
 
 Also, there are some basic tests built in to the library, so please test your code before you try to get it merged (_just to make sure everything is backwards compatible_). Use `npm run test` to do this.
 
-# Documentation
+## Options
+
+You can also run `parseGPX()` with custom options for more control over the parsing process. See [options.ts](./lib/options.ts) for more details.
```

Comment:
The individual options need to be explained either in the README or the options.ts file you link to. If you do it in the file, comments on each line explaining each option should suffice for now.